runtime/test-runtime/src/genesismap.rs

```diff
@@ -67,6 +67,8 @@ impl GenesisConfig {
 			parachains: None,
 			registrar: None,
 			vesting: None,
+			authority_discovery: None,
```

Comment:
We don't need this file anymore. This handled by `chain_spec`.

Reply:
There is a problem of dependency then because chain_spec is in polkadot-test-service and polkadot-test-service depends on polkadot-test-runtime

Reply:
I don't understand what you are saying?

Reply:
I understood this: we don't need the file `runtime/test-runtime/src/genesismap.rs` because it is handled by the file `polkadot-test-service/src/chain_spec.rs`.

So I tried to remove this file and replace its use in [here](https://github.com/paritytech/polkadot/blob/master/runtime/test-runtime/client/src/lib.rs#L92) by [this one](https://github.com/paritytech/polkadot/blob/cecton-polkadot-test-runtime/polkadot-test-service/src/chain_spec.rs#L32) but it is not possible because polkadot-test-service uses test-runtime in its dependency and I would need to add polkadot-test-service to test-runtime's dependencies.